show/main.py

```diff
@@ -878,25 +878,30 @@ def platform():
 @platform.command()
 def summary():
     """Show hardware platform information"""
-    username = getpass.getuser()
-
-    PLATFORM_TEMPLATE_FILE = "/tmp/cli_platform_{0}.j2".format(username)
-    PLATFORM_TEMPLATE_CONTENTS = "Platform: {{ DEVICE_METADATA.localhost.platform }}\n" \
-                                 "HwSKU: {{ DEVICE_METADATA.localhost.hwsku }}\n" \
-                                 "ASIC: {{ asic_type }}"
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    data = config_db.get_table('DEVICE_METADATA')
 
-    # Create a temporary Jinja2 template file to use with sonic-cfggen
-    f = open(PLATFORM_TEMPLATE_FILE, 'w')
-    f.write(PLATFORM_TEMPLATE_CONTENTS)
-    f.close()
+    try:
+        platform = data['localhost']['platform']
```

Comment:
I think the ground truth for 'platform' is in /host/machine.conf instead of ConfigDB.

Reply:
In latest commit, I now obtain information via sonic_platform module where possible, which has logic to directly parse /host/machine.conf.